check-log/lib/check-log.go

```diff
@@ -220,7 +236,7 @@ func run(args []string) *checkers.Checker {
 	return checkers.NewChecker(checkSt, msg)
 }
 
-func (opts *logOpts) searchLog(logFile string) (int64, int64, string, error) {
+func (opts *logOpts) searchLog(ctx context.Context, logFile string) (int64, int64, string, error) {
 	stateFile := getStateFile(opts.StateDir, logFile, opts.origArgs)
```

Comment:
Maybe we should return without scanning when ctx has been already canceled at start of this method.